processor/integrations/integrations.go

```diff
@@ -166,7 +166,7 @@ func GetTransformerURL() string {
 
 // GetDestinationURL returns node URL
 func GetDestinationURL(destType string) string {
-	destinationEndPoint := fmt.Sprintf("%s/v0/%s", destTransformURL, strings.ToLower(destType))
+	destinationEndPoint := fmt.Sprintf("%s/v0/destinations/%s", destTransformURL, strings.ToLower(destType))
```

Comment:
Is the destination URL changed?

Reply:
https://rudderlabs.slack.com/archives/C01HTT66UMB/p1668245015817599?thread_ts=1668237366.021429&cid=C01HTT66UMB